src/Arcus.BackgroundJobs.DataBricks/DatabricksInfoProvider.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Azure.Databricks.Client;
+
+namespace Arcus.BackgroundJobs.Databricks
+{
+    /// <summary>
+    /// Provides dev-friendly access to the Databricks instance.
+    /// </summary>
+    internal class DatabricksInfoProvider : IDisposable
```

Comment:
Might be good to have unit tests with a mocked `DatabricksClient` to ensure that the job processing matches the correct runs, and we report metrics for all required jobs.

Reply:
😢 Yes, but the `DatabricksClient` is sealed so except if you want me to write dynamic proxies or so (have to look some up), I can't mock this. We could use `IJobsApi` interface (which is the `.Jobs` property of the `DatabricksClient`) but that would make this provider less user-friendly though. 🤔 .

Reply:
It's ok, parked it in #41